### PR TITLE
Fixed #25659 -- Added missing support for MySQL 5.6.1 to GeoDjango DB functions.

### PR DESCRIPTION
**Pull request by sir-sigurd**

### CODE REVIEW

File: tests/gis_tests/geoapp/test_functions.py [hunk 1]
```diff
@@ -431,14 +430,12 @@ def test_diff_intersection_union(self):
...
+            self.assertTrue(c.mpoly.difference(geom).equals(c.difference))
+            if not (spatialite or mysql):
                 self.assertEqual(c.mpoly.intersection(geom), c.intersection)
             # Ordering might differ in collections
-            self.assertSetEqual(set(g.wkt for g in c.mpoly.sym_difference(geom)),
-                                set(g.wkt for g in c.sym_difference))
-            self.assertSetEqual(set(g.wkt for g in c.mpoly.union(geom)),
-                                set(g.wkt for g in c.union))
+            self.assertTrue(c.mpoly.sym_difference(geom).equals(c.sym_difference))
+            self.assertTrue(c.mpoly.union(geom).equals(c.union))
```
Comment (claudep) [inline tests/gis_tests/geoapp/test_functions.py]: If I understand this change, it is not sensible to ordering any more? If yes, then the comment above should be removed.

File: tests/gis_tests/geoapp/test_functions.py [hunk 2]
```diff
@@ -377,8 +377,7 @@ def test_sym_difference(self):
         qs = Country.objects.annotate(sym_difference=functions.SymDifference('mpoly', geom))
         for country in qs:
             # Ordering might differ in collections
-            self.assertSetEqual(set(g.wkt for g in country.mpoly.sym_difference(geom)),
-                                set(g.wkt for g in country.sym_difference))
+            self.assertTrue(country.mpoly.sym_difference(geom).equals(country.sym_difference))
```
Comment (claudep) [inline tests/gis_tests/geoapp/test_functions.py]: Same remark about ordering here

**Comment by claudep** [comment]
buildbot, test on precise

File: tests/gis_tests/geoapp/test_functions.py (hunk 1 above)
Comment (sir-sigurd) [inline tests/gis_tests/geoapp/test_functions.py]: yup, thanks

**Comment by claudep** [comment]
Thanks, merged in 7127eb287f.

**Pull request closed without merging**